llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -17,6 +17,24 @@ include "mlir/Interfaces/TilingInterface.td"
 include "mlir/Interfaces/ViewLikeInterface.td"
 
 
+def IREELinalgExt_DoNotDCEOperandsOp : 
```

Comment:
This op already exists in IREE: `util.do_not_optimize`. We should be able to use it here.

Reply:
We need something that will work upstream